### PR TITLE
feat(billing.autorenew.bulk): display eligible services

### PR DESCRIPTION
**Pull request by marie-j**
## Display eligible services

MANAGER-3291

### CODE REVIEW

**Comment by ovh-cds** [comment]
CDS Report build-manager#2734.0 ✘  
* Stage 1
  * Build dependencies and inject Sentry ✔ 
* Stage 2
  * Build manager CA ✘  
  * Build manager EU ✘  
  * Build manager US ✘  
  * Tests ✔

File: client/app/billing/autoRenew/bulk/bulk.html [hunk 1]
```diff
@@ -50,6 +50,21 @@
                 </ul>
             </li>
         </ul>
+        <div class="mb-4">
+            <button
+                type="button"
+                class="oui-link"
+                data-ng-click="$ctrl.expand.available = !$ctrl.expand.available"
+                data-ng-bind="($ctrl.expand.available ? 'billing_autorenew_bulk_available_less' : 'billing_autorenew_bulk_available_more') | translate">
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/bulk/bulk.html]: Personal preference:
```suggestion
                data-translate="{{:: $ctrl.expand.available ? 'billing_autorenew_bulk_available_less' : 'billing_autorenew_bulk_available_more' }}">
```

File: client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -13,5 +13,8 @@
   "billing_autorenew_bulk_service": "(1 service)",
   "billing_autorenew_bulk_services": "({{ count }} services)",
   "billing_autorenew_bulk_more": "Voir plus",
-  "billing_autorenew_bulk_less": "Voir moins"
+  "billing_autorenew_bulk_less": "Voir moins",
+  "billing_autorenew_bulk_available_more": "Voir le détail",
+  "billing_autorenew_bulk_available_less": "Masquer le détail",
+  "billing_autorenew_bulk_available": "des services compatibles"
```
Comment (FredericEspiau) [inline client/app/billing/autoRenew/bulk/translations/Messages_fr_FR.json]: This is not translatable, could you make a whole sentence instead ?

**Comment by jleveugle** [comment]
Hello @marie-j,
Your pull request is now closed due to inactivity.
Feel free to re-open it if you have any update.
Anna Bot for OVH UX <3

**Pull request merged**